packages/manager/modules/nutanix/src/cluster.class.js

```diff
@@ -0,0 +1,15 @@
+import { get } from 'lodash';
+
+export default class Cluster {
+  constructor({ serviceName, targetSpec, status }) {
+    Object.assign(this, {
+      serviceName,
+      targetSpec,
+      status,
+    });
+  }
+
+  getNodes() {
+    return get(this, 'targetSpec.nodes');
```

Comment:
To avoid lodash `get`

```suggestion
    return this.targetSpec?.nodes;
```

Reply:
Probably, return `[]` if `this.targetSpec.nodes` is not available?